config/webpack/webpack.config.js

```diff
@@ -8,21 +14,77 @@ const basedir = join(__dirname, '../..');
 
 const config = require('../config');
 
+const isDevelopment = process.env.NODE_ENV === 'development';
 const isProduction = process.env.NODE_ENV === 'production';
 
+const ifDevelopment = (...items) => {
+  return isDevelopment ? items : [];
+};
+
+const ifProduction = (...items) => {
+  return ifProduction ? items : [];
+};
+
+const styleLoaders = [
+  isProduction ? {
+    loader: MiniCssExtractPlugin.loader
+  } : {
+    loader: 'style-loader'
+  },
+  {
+    loader: 'css-loader',
+    options: { importLoaders: 1 }
+  },
+  {
+    loader: 'postcss-loader',
+    options: {
+      ident: 'postcss',
+      plugins: [
+        require('autoprefixer')()
+      ]
+    }
+  }
+];
+
 module.exports = merge(shims, {
   mode: isProduction ? 'production' : 'development',
+  devtool: isProduction ? 'source-map' : 'cheap-module-eval-source-map',
   entry: require.resolve('./entries/main'),
   output: {
-    path: join(basedir, 'public/assets')
+    path: join(basedir, 'public/assets'),
```

Comment:
I'm not 100% sure but I think Windows is happier with `join(basedir, 'public', 'assets')`

Reply:
Windows also accepts forward slashes in most places, but I switched it to use multiple args in `join()` anyway.

Reply:
Cheers! This is one thing I've meant to double-check someday, it's just memory-rule I have from somewhere. I haven't heard anyone try run TR on Windows for a long time so it's not a biggie to keep guarding compatibility. :-)